integration-test/tests/login-form.macos.spec.js

```diff
@@ -99,17 +99,10 @@ test.describe('Auto-fill a login form on macOS', () => {
         server.close()
     })
     test.describe('without getAvailableInputTypes API', () => {
-        test.skip('with in-page HTMLTooltip', async ({page}) => {
+        test('with in-page HTMLTooltip', async ({page}) => {
             await testLoginPage(page, server)
         })
         test.describe('with overlay', () => {
-            test('with autoprompt', async ({page}) => {
```

Comment:
I've removed the autoprompt on macOS because it wasn't really helping the user experience, but creating subtle issues and performance hit (because of the endless polling).